apps/cookbook/src/app/showcase/data-table-showcase/data-table-showcase.component.html

```diff
@@ -28,10 +12,39 @@ <h2>Data table with minimal data</h2>
   </cookbook-example-viewer>
 
   <h2>Data table wrapped with Kirby Card</h2>
-  <p>Data table wrapped with card and with selectable rows.</p>
+  <p>Data table wrapped with card, with selectable rows.</p>
```

Comment:
Consider wording and grammar:
```suggestion
  <p>Data table wrapped in a card with selectable rows.</p>
```